docs/lib/graphqlapi/fragments/native_common/authz/common.md

```diff
@@ -18,6 +18,10 @@ Amazon Cognito Identity Pools allows you to use credentials from AWS IAM in your
 
 If you are using a 3rd party OIDC provider you will need to configure it and manage the details of token refreshes yourself. 
 
+### Lambda Authorizer
+
+You can implement your own API authorization logic using an AWS Lambda function. To add a Lambda authorizer as an authentication mode for you API, please see the Settings section of the AppSync console. You will need to manage the details of token refreshes in your application code yourself. 
+
```

Comment:
should we also add a code snippet of an example of Lambda here?

Reply:
My plan was to link to the AppSync docs for Lambda auth here (once they become publicly available) and those would give customers more info on configuring a Lambda authorizer. We'll be adding CLI support in the near future, which would generate a basic authorizer for them.